src/main/deploy/deploy

```diff
@@ -1,3 +1,2 @@
-Files placed in this directory will be deployed to the RoboRIO into the
```

Comment:
? Mistaken deleted line?

Reply:
Probably? It was done in 66d7809 (pivot offsets), which wouldn't have a reason to delete that line (especially since it left the other lines in the file).